modules/consignment-api/database.tf

```diff
@@ -0,0 +1,59 @@
+resource "random_password" "password" {
+  length  = 16
+  special = false
+}
+
+resource "aws_db_subnet_group" "consignment_api_subnet_group" {
+  name       = "tdr-${var.environment}"
+  subnet_ids = var.private_subnets
+
+  tags = merge(
+    var.common_tags,
+    map(
+      "Name", "user-db-subnet-group-${var.environment}"
+    )
+  )
+}
+
+resource "random_string" "snapshot_prefix" {
+  length  = 4
+  upper   = false
+  special = false
+}
+
+resource "aws_rds_cluster" "consignment_api_database" {
+  cluster_identifier_prefix = "consignment-api-db-${var.environment}"
+  engine                    = "aurora-mysql"
```

Comment:
I thought this was going to be Postgres like in the prototype. Did that decision change?

Reply:
We were going to use serverless but that didn't work with Flyway's free version so rather than changing the code back to postgres, I changed it to non-serverless mysql. We can always change it later.